saltysimulations/salty-747
# feat(fmc): t/d indication in progress page + unit conversion fix

**Testing instructions for unit coversion fix:**

- Spawn somewhere and have a look at FMC Take Off Page
- Same gross weight amount should show up like on EICAS Fuel or Upper EICAS
- If Unit in SALTY OPTION is changed from KG to LBS or LBS to KG the value of gross weight should change accordingly

**Testing instructions for T/D Progress Page:**

- Plan a Flight with a distance > 250 NM
- Set up a Flight Plan with Arrival with Runway and Approach Route of your choice selected (Green T/D should be visible on MFD)
- Fly like usual
- at 200 NM before T/D remaing distance should show up like on screenshot
- at 0.5 NM before T/D begin it should change to NOW and then disappear
